voluncheer/profiles/urls.py

```diff
@@ -2,16 +2,35 @@
 from django.urls import path
 
 from profiles.views.home import home
-from profiles.views.profile import *
+from profiles.views.profile import ProfileView
+from profiles.views.profile import profile_update
 
 urlpatterns = [
     path("", home, name="home"),
-    path(r'^profile/$', ProfileView.as_view(), name="profile"),
-    path("organizations/", include(([
-        # Here we can place organization only URLs.
-    ], "profiles"), namespace="organizations")),
-    path("volunteers/", include(([
-        # Here we can place volunteer only URLs.
-    ], "profiles"), namespace="volunteers")),
-    path(r'^profile/update/$', form_valid, name="profile_update"),
-]
\ No newline at end of file
+    path(r"^profile/$", ProfileView.as_view(), name="profile"),
+    path(
```

Comment:
We can actually just remove this since it isn't doing anything. I mistakenly left it in.

Reply:
Sure